source/_integrations/simplepush.markdown

```diff
@@ -13,43 +14,15 @@ ha_integration_type: integration
 
 The `simplepush` platform uses [Simplepush](https://simplepush.io/) to delivery notifications from Home Assistant to your Android device. Unlike similar apps the Simplepush app requires no registration.
 
-To add Simplepush to your installation, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-notify:
-  - name: NOTIFIER_NAME
-    platform: simplepush
-    device_key: ABCDE
-```
-
-{% configuration %}
-  name:
-    description: Setting the optional parameter `name` allows multiple notifiers to be created. The default value is `notify`. The notifier will bind to the service `notify.NOTIFIER_NAME`.
-    required: false
-    type: string
-  device_key:
-    description: The device key of your device.
-    required: true
-    type: string
-  event:
-    description: The event for the events.
-    required: false
-    type: string
-  password:
-    description: The password of the encryption used by your device.
-    required: inclusive
-    type: string
-  salt:
-    description: The salt used by your device.
-    required: inclusive
-    type: string
-{% endconfiguration %}
+{% include integrations/config_flow.md %}
 
 To test if the service works, just send a message with `curl` from the command-line.
 
 ```bash
 curl 'https://api.simplepush.io/send/device_key/title/message'
 ```
+## Notifications
+
+Simplepush can send a notification by calling the [`notify` service](/integrations/notify/). You can sepcify the `event` under the `data` key. This will override the event configured in YAML when imported.
```

Comment:
```suggestion
Simplepush can send a notification by calling the [`notify` service](/integrations/notify/). You can specify the `event` under the `data` key. This will override the event configured in YAML when imported.
```